### ISSUE TITLE
Question about evaluation

### ISSUE THREAD

**Issue by akonstantinou** [VisualComputingInstitute/triplet-reid#23]
Hello, i wish to evaluate your pre-trained model with market dataset. I created the embendings of the test set in a csv file. How can i use the csv file to calculate mAP and rank-1 statistics? Do i have to convert the csv to h5? If yes then how? Thank you.

**Comment by lucasb-eyer** [comment]
Hi,

Indeed, the generated csv file is not directly useable by the evaluation code. (The `trinet_embed.py` is old code which wasn't meant to be used with the new code.)
You are correct that you would need to convert those to HDF5 files, and then you should be able to evaluate. We don't have code for this, as it was never a use-case for us, but you could write a small python script which loads the csv file and writes a HDF5 file. You could take `embed.py` as a starting point to see how we write the HDF5 file.

Since we don't plan on providing such a tool ourselves, and Issue #16 keeps track of the need to provide pre-trained models for the "new" code (I forgot about it!), I'm closing this issue and considering it a duplicate of #16, although it technically isn't.

Hope that helps.
